src/main/java/org/truffleruby/language/backtrace/Backtrace.java

```diff
@@ -22,15 +22,21 @@
 
 public class Backtrace {
 
+    private final Node location;
```

Comment:
I was thinking to get the node from the first activation, but this is cleaner and does not rely on the logic of the CallStackManager.